.github/workflows/gerald-pr.yml

```diff
@@ -5,16 +5,27 @@ jobs:
   gerald:
     runs-on: ubuntu-latest
     steps:
+      - name: Get All Changed Files
+        uses: jaredly/get-changed-files@absolute
+        id: changed
+        with:
+          format: 'json'
+      - uses: allenevans/set-env@v2.0.0
+        with:
+          ALL_CHANGED_FILES: '${{ steps.changed.outputs.added_modified }}'
```

Comment:
Is this unrelated? Just a change to properly know what files changed?

Reply:
Ah, so Gerald had stopped running in this repo - needed to update it so that it would run properly!